rpcserver.go

```diff
@@ -1937,6 +1938,13 @@ func (r *rpcServer) parseOpenChannelReq(in *lnrpc.OpenChannelRequest,
 		channelFeeRate = &in.FeeRate
 	}
 
+	// Ensure that the remove channel reserve does not exceed the channel
+	// capacity.
+	if remoteChanReserve >= localFundingAmt {
```

Comment:
Should we instead already check for the 20% here? Otherwise we might pass this check but then fail in the manager's verification?